chromium_src/chrome/common/url_constants.cc

```diff
@@ -45,22 +45,22 @@ const char kChromeReleaseNotesURL[] =
 
 const char kChromeHelpViaKeyboardURL[] =
 #if defined(OS_CHROMEOS)
-#if defined(GOOGLE_CHROME_BUILD)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
     "https://support.brave.com/";
 #else
     "https://support.brave.com/";
-#endif  // defined(GOOGLE_CHROME_BUILD)
+#endif  // BUILDFLAG(GOOGLE_CHROME_BRANDING)
 #else
     "https://support.brave.com/";
 #endif  // defined(OS_CHROMEOS)
 
 const char kChromeHelpViaMenuURL[] =
 #if defined(OS_CHROMEOS)
-#if defined(GOOGLE_CHROME_BUILD)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
```

Comment:
why do we have these buildflags at all in our own file? That makes no sense.

Reply:
it especially makes no sense because the url is the same

Reply:
apparently this file is auto generated so these changes won't work anyway

Reply:
Removed changed to this file since they make no difference. If we want to clean it up it can be done as a separate effort.